### PR TITLE
feat: lazy initialize $recaptcha on execute

### PR DESCRIPTION
**Pull request by pi0**

### CODE REVIEW

File: lib/plugin.js [hunk 1]
```diff
@@ -1,44 +1,63 @@
...
+      throw new Error('ReCaptcha error: No key provided')
     }
 
+    this._ready = false
+
     this.siteKey = siteKey
+    this.hideBadge = hideBadge
+  }
+
+  init () {
```
Comment (manniL) [inline lib/plugin.js]: Maybe 
```suggestion
  async init () {
```

Or do you try to avoid that? :)

Comment (mvrlin) [reply lib/plugin.js]: @manniL Do you have an idea how to make it the better way?

Comment (pi0) [reply lib/plugin.js]: This makes extra code after transpilation also extra wrapper promise. Anyway we need internal `new Promise`

Comment (manniL) [reply lib/plugin.js]: @pi0 Wondering why we'd need a `new Promise` inside? return values from async functions are always a Promise I thought 🤔

Comment (manniL) [reply lib/plugin.js]: @pi0 Ah, because of line 32. nvm

**Pull request merged**